rest/taskana-rest-spring/src/main/java/pro/taskana/user/jobs/UserInfoRefreshJob.java

```diff
@@ -0,0 +1,127 @@
+package pro.taskana.user.jobs;
```

Comment:
In general I would prefer to have some more debug logging in such kind of jobs to support explaining weird behaviour on the customer machine... ;-)